torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -115,10 +145,10 @@ class AutoAugment(_AutoAugmentBase):
         "ShearX": (lambda num_bins, image_size: torch.linspace(0.0, 0.3, num_bins), True),
         "ShearY": (lambda num_bins, image_size: torch.linspace(0.0, 0.3, num_bins), True),
         "TranslateX": (lambda num_bins, image_size: torch.linspace(0.0, 150.0 / 331.0 * image_size[1], num_bins), True),
-        "TranslateY": (lambda num_bins, image_size: torch.linspace(0.0, 150.0 / 331.0 * image_size[1], num_bins), True),
+        "TranslateY": (lambda num_bins, image_size: torch.linspace(0.0, 150.0 / 331.0 * image_size[0], num_bins), True),
         "Rotate": (lambda num_bins, image_size: torch.linspace(0.0, 30.0, num_bins), True),
         "Brightness": (lambda num_bins, image_size: torch.linspace(0.0, 0.9, num_bins), True),
-        "Color": (lambda num_bins, image_size: torch.linspace(0.0, 0.9, num_bins), True),
+        "Saturation": (lambda num_bins, image_size: torch.linspace(0.0, 0.9, num_bins), True),
```

Comment:
Though your proposed names describe more accurately what the ops do, we should revert all the name changes to their original values. This is to align with all the papers that use the same terminology. This originally came from the [AutoAugment](https://arxiv.org/pdf/1805.09501.pdf) paper at page 13.